core/templates/dev/head/editor/StateResponses.js

```diff
@@ -306,6 +306,38 @@ oppia.controller('StateResponses', [
       trainingDataService.initializeTrainingData(
         explorationId, currentStateName);
     };
+    $scope.suppressDefaultAnswerGroupWarnings = function() {
+      var interactionId = $scope.getCurrentInteractionId();
+      if (interactionId === 'MultipleChoiceInput') {
+        var choices = $scope.getAnswerChoices().length;
+        var suppressAnswer = responsesService.getAnswerGroups();
+        var defaultAnswers;
+        // For choice length, have indices 0 through choice length - 1
+        for (var x = 0; x < choices; x++) {
+          if (suppressAnswer['' + x + ''] === undefined) {
+            // Return false if no object exists for the index
+            /* Note that multiple choice only allows for
```

Comment:
I'm not sure I understand what you're saying here. When I create a multiple choice answer group, I can only choose the rule "Equal To." After I've create the original answers, I can try to add another answer that coincides with the same choice but Oppia throws an error.